Src/Microsoft.Dynamic/Generation/CompilerHelpers.cs

```diff
@@ -857,19 +856,19 @@ private sealed class DebuggableCodeRewriter : DynamicExpressionVisitor {
         #region Factories
 
         public static BigInteger CreateBigInteger(bool isNegative, byte[] data) {
-            return new BigInteger(CreateBigInt(isNegative, data));
+            return CreateBigInt(isNegative, data);
         }
 
-        public static BigInt CreateBigInt(int value) {
-            return (BigInt)value;
+        public static BigInteger CreateBigInt(int value) {
```

Comment:
I know these are public, but I'm wondering if we should still rename them to `CreateBigInteger`...

Reply:
Not sure. Do you still want me to do this change?

Reply:
I would like to mark this one as Obsolete and add `CreateBigInteger` and rename all internal and IronPython usages to use `CreateBigInteger`. We can do as a separate PR if you don't want to do it in this one.